examples/multiple-apps.md

```diff
@@ -0,0 +1,85 @@
+# Control the delivery process of multiple applications
+
+> Note: You need to install the definitions in the [example directory](https://github.com/kubevela/workflow/tree/main/examples/definitions) first to run the example.
```

Comment:
please also add the commands to tell how to install these definitions.

I suggest these definitions to be built-in definitions installed along with addons.